static/js/stream_create.js

```diff
@@ -271,11 +275,12 @@ exports.show_new_stream_modal = function () {
     });
 };
 
-$(function () {
-    $('body').on('change', '#user-checkboxes input, #make-invite-only input', update_announce_stream_state);
+exports.create_handlers_for_users = function (container) {
+    // container should be $('#people_to_add')...see caller to verify
+    container.on('change', '#user-checkboxes input', update_announce_stream_state);
 
     // 'Check all' and 'Uncheck all' visible users
-    $(document).on('click', '.subs_set_all_users', function (e) {
+    container.on('click', '.subs_set_all_users', function (e) {
         $('#user-checkboxes .checkbox').each(function (idx, li) {
             if  (li.style.display !== "none") {
```

Comment:
It is my first time reviewing, so please bear with me, I think we can probably clean this up 
```javascript
$('#user-checkboxes .checkbox')
```
since this is used two times, here and in unset part. Do let me know whether I am thinking in the right direction.

Reply:
Also, here https://github.com/zulip/zulip/blob/eb0da20b7852711083a510a0d2eb731be0379d47/static/js/stream_create.js#L375
we need to use 'container' you declared.

Reply:
@Kumar-Tarun Oh yeah, that's probably a good idea.